isp/test/api_audit_operations_for_organization_test.go

```diff
@@ -0,0 +1,40 @@
+/*
+iStreamPlanet Channels API
+
+Testing AuditOperationsForOrganizationApiService
+
+*/
+
+// Code generated by OpenAPI Generator (https://openapi-generator.tech);
+
+package isp
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+	openapiclient "github.com/GIT_USER_ID/GIT_REPO_ID/isp"
+)
+
+func Test_isp_AuditOperationsForOrganizationApiService(t *testing.T) {
+
+	configuration := openapiclient.NewConfiguration()
+	apiClient := openapiclient.NewAPIClient(configuration)
+
+	t.Run("Test AuditOperationsForOrganizationApiService GetOrgChannelTimeline", func(t *testing.T) {
+
+		t.Skip("skip test")  // remove to run test
```

Comment:
All of the generated tests are commented out.  Is this the way we want it?

Reply:
This is how they are generated, and I'm not entirely convinced generated tests on generated code are even particularly useful. I'll add a task for writing some more useful sanity-check style tests and setting up the repo to run them properly on PRs in the future. We just need to figure out the auth I think.